flink-libraries/flink-table/src/test/java/org/apache/flink/api/java/table/test/StringExpressionsITCase.java

```diff
@@ -40,46 +40,6 @@ public StringExpressionsITCase(TestExecutionMode mode) {
 		super(mode);
 	}
 
-	@Test(expected = CodeGenException.class)
```

Comment:
Agreed, we should not have end-to-end tests for all scalar functions. But it would be good to tests at least some representative functions, IMO.